include/aspect/particle/property/reference_position.h

```diff
@@ -0,0 +1,88 @@
+/*
+ Copyright (C) 2015 - 2019 by the authors of the ASPECT code.
```

Comment:
This is actually nice to have, deal.II already asked to remove the reference position from inside the particles and instead store it as an otional property, like you do here (I know this was meant as a postprocessing property, but eventually this will probably become the main storage location for the reference position).

Reply:
Ok, then I'll keep it in the PR.